python/ray/dag/dag_node_operation.py

```diff
@@ -18,12 +23,22 @@ class _DAGNodeOperationType(Enum):
     COMPUTE = "COMPUTE"
     WRITE = "WRITE"
 
+    def __str__(self):
+        if self == _DAGNodeOperationType.READ:
+            return "R"
```

Comment:
Why not return the full string?

Reply:
This is used in visualization only, using a shorter form for conciseness.

Renamed to `short_str()` and added docstring.